client/src/components/About.tsx

```diff
@@ -15,39 +23,93 @@ export const About: React.StatelessComponent = (): JSX.Element => (
         </Link>
         <div className="center">
             <h4>About</h4>
-            <h4><a href="https://github.com/oscarso2000/mycourseindex">Source code on Github</a></h4>
-            <p> The authors of the MyCourseIndex project are: </p>
 
-            <div className="item">
-                <img src={Magd} />
-                <span className="caption"><a href="https://github.com/bayoumi17m">Magd Bayoumi (mb2363)</a></span>
+            <p>MyCourseIndex was initially a project started for CS/INFO 4300: Language and Information that acts as an essential search engine for Cornell students and their courses. This search gathers all information from Piazza posts to Textbook and Notes Resources and returns valid results for the student to utilize. It is now being maintained and improved by members of <a href="https://cornelldata.science">Cornell Data Science Project Team</a>.</p>
+
+            <div className="centertext">
+                <p> <b>Co-Founders:</b> </p>
             </div>
-            <div className="item">
-                <img src={Jenna} />
-                <span className="caption"><a href="https://github.com/jek343">Jenna Kressin (jek343)</a></span>
+            <div className="innerimage">
+                <div className="item">
+                    <img src={Magd} />
```

Comment:
It's not something serious but I'm getting this javascript warning in my terminal. I think that we should add an alt text here just to follow the convention !
<img width="1064" alt="Screen Shot 2020-09-19 at 1 05 35 PM" src="https://user-images.githubusercontent.com/59305253/93658677-d1236a00-fa78-11ea-8690-79fc6c63ad03.png">